dashboard/bin/queue_pd_reminder_emails

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+require_relative '../../lib/cdo/only_one'
+exit unless only_one_running?(__FILE__)
+
+require_relative '../config/environment'
+
+def queue_pd_reminder_emails
```

Comment:
what will the time difference be between when these emails are queued and when they are sent? what happens if some of the criteria for sending this email change during that time? (if the principal submits their approval or the status changes?)

Reply:
Great question.  At first I was thinking of doing it manually the afternoon before, but you're right about potential state changes.  For now, I think about 30 minutes before seems like a good idea.. enough time to disable the cron job that sends all queued emails if it turns out we queued up an obviously wrong set of emails.  The longer term plan is to generate these emails immediately before they are dequeued, if not just sending them immediately once we're confident in the process.